src/components/Navbar/Navbar.component.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+import './Navbar.styles.css';
+
+function Navbar() {
+  return <></>;
```

Comment:
**Question**: What is the idea of this component?

Reply:
My idea is make something like the navbar of youtube with a hamburguer menu on the left side and when its a mobile device on the bottom; in the hamburguer menu im thinking put the access to favorite section like in the example of the final app.